Makefile

```diff
@@ -167,17 +167,20 @@ vet:
 push-olm-manifests: verify-olm-manifests
 	./hack/push-olm-manifests.sh $(OLM_PACKAGE_ORG) metering $(OLM_PACKAGE_VERSION)
 
-verify: verify-codegen verify-olm-manifests verify-helm-templates fmt vet
+verify: verify-codegen verify-ocp-olm-manifests verify-upstream-olm-manifests verify-helm-templates fmt vet
 	@echo Checking for unstaged changes
 	# validates no unstaged changes exist in $(VERIFY_FILE_PATHS)
 	git diff --stat HEAD --ignore-submodules --exit-code -- $(VERIFY_FILE_PATHS)
 
 verify-helm-templates:
 	helm template ./charts/openshift-metering > /dev/null
 
-verify-olm-manifests: metering-manifests
```

Comment:
I feel like we should just have a single target which verifies both. Any reason you did them separately?

Reply:
nope, was just doing this on the fly and didn't consider that - I'll update that